plotly/tests/test_core/test_tools/test_Quiver.py

```diff
@@ -0,0 +1,223 @@
+import plotly
+from plotly.graph_objs import *
```

Comment:
:cow2: You shouldn't use the import `*` notation within our own packages.

Instead just import the things you need directly.